### PR TITLE
Update phpunit-core for the moved vendor directory

### PR DESCRIPTION
**Pull request by bojanz**
The vendor dir has moved to root. This should make phpunit-core work again.

### CODE REVIEW

File: runners/phpunit-core/script.sh [hunk 1]
```diff
@@ -11,4 +11,4 @@ ARGS=( $DRUPAL_TI_PHPUNIT_ARGS )
 
 # Run core tests
 cd core
```
Comment (slashrsm) [inline runners/phpunit-core/script.sh]: Could we remove this cd instead?

Comment (bojanz) [inline runners/phpunit-core/script.sh]: We can, but then we need to add a "-c core" to the phpunit invocation. Thought that might interfere with "${ARGS[@]}", though it's not very likely.

Comment (LionsAd) [inline runners/phpunit-core/script.sh]: I am fine with the cd. Thanks for the patch!

**Comment by LionsAd** [comment]
Thanks for the PR!

**Comment by joshuataylor** [comment]
Do you mind tagging this as a release? 1.4.1 is a little old now (https://packagist.org/packages/lionsad/drupal_ti)

**Comment by LionsAd** [comment]
1.4.2 has been tagged.

**Pull request merged**